_drgn.pyi

```diff
@@ -1110,6 +1110,17 @@ class Thread:
 
     tid: Final[int]
     """Thread ID (as defined by :manpage:`gettid(2)`)."""
+    name: Optional[str]
+    """
+    Thread name, or ``None`` if unknown.
+    See `PR_SET_NAME
+    <https://man7.org/linux/man-pages/man2/prctl.2.html#:~:text=PR_SET_NAME>`_
+    and `/proc/[pid]/comm
+    <https://man7.org/linux/man-pages/man5/proc.5.html#:~:text=%2Fproc%2F%5Bpid%5D%2Fcomm>`_.
+    .. note::
```

Comment:
These links have moved since the original PR. Also, Sphinx needs some blank lines or it gets mad.

```suggestion
    Thread name, or ``None`` if unknown.

    See `PR_SET_NAME
    <https://man7.org/linux/man-pages/man2/PR_SET_NAME.2const.html>`_ and
    `/proc/pid/comm
    <https://man7.org/linux/man-pages/man5/proc_pid_comm.5.html>`_.

    .. note::

```